src/module.c

```diff
@@ -9763,6 +9763,10 @@ ValkeyModuleString *VM_GetModuleUserACLString(ValkeyModuleUser *user) {
  * The returned string must be released with ValkeyModule_FreeString() or by
  * enabling automatic memory management. */
 ValkeyModuleString *VM_GetCurrentUserName(ValkeyModuleCtx *ctx) {
+    if (ctx == NULL || ctx->client == NULL || ctx->client->user == NULL) {
```

Comment:
let's also check the name in case.
```suggestion
    if (ctx == NULL || ctx->client == NULL || ctx->client->user == NULL || ctx->client->user->name == NULL) {
```